tests/__data__/pre2017.json

```diff
@@ -1,11 +1,7 @@
 [
   {
     "year": 2016,
-    "statistics": {
-      "number_of_organizations": 17,
-      "number_of_participants": 1340,
-      "total_tasks_completed": 6418
-    },
+    "statistics": {},
```

Comment:
This seems potentially unrelated?

Reply:
Yep. I just find out it is another bug. Created an issue for that https://github.com/coala/gci-leaders/issues/162